lib/fs/casefs.go

```diff
@@ -397,18 +397,23 @@ func (f *caseFilesystem) checkCaseExisting(name string) error {
 type defaultRealCaser struct {
 	cache *caseCache
 	fs    Filesystem
-	mut   sync.Mutex
 }
 
-type caseCache = lru.TwoQueueCache[string, *caseNode]
+type caseCache struct {
+	*lru.TwoQueueCache[string, *caseNode]
```

Comment:
nit: I don't think there's any need for this to be a pointer, we do already handle `caseCache` instances as pointers (and must to avoid copying the mutex). It was a pointer before the referenced change, but I don't see any reason for it to be a pointer there/in history either.

Reply:
You can dereference the pointer when assigning it to `TwoQueueCache` by preceding it with `*`.